locales/base/translation.json

```diff
@@ -2449,6 +2449,11 @@
       "title": "Earn points effortlessly",
       "description": "Jump into a new way to earn! Your {{appName}} actions now earn points. Start earning today and unlock awesome benefits.",
       "cta": "Start your points journey"
+    },
+    "disclaimer": {
+      "learnMoreCta": "<0>Learn more</0> about {{appName}} Points",
+      "body": "<0>What are Points?</0>\nPoints are {{appName}}'s loyalty program. They're awarded to users when you use our app.\n\n<0>What are they good for?</0>\nYou collect them. They can’t be spent or traded or transferred.\n\n<0>Why are they in my wallet?</0>\nBecause they’re tokens. On chain, digital bits of happiness you earn by using the app.",
```

Comment:
I'm curious, how could they be in the wallet? Unless imported

Reply:
i'm going to merge this and update the copy separately once it's ready (i think there will be copy changes for swaps too, so can do those together)